src/SeqViz/Circular/AnnotationsViewer.tsx

```diff
@@ -14,9 +42,9 @@ import { COLOR_BORDER_MAP, darkerColor } from "../../utils/colors";
  *
  * @type {Function}
  */
-export default class Annotations extends React.PureComponent {
+export default class AnnotationsViewer extends React.PureComponent<AnnotationsViewerProps> {
```

Comment:
I think I get the intent of this: `Annotations` like the data format/interface, versus the thing that renders them in the Linear/Circular viewers. But if keeping those separate I think I'd name the data format/interface as `IAnnotation`, `IIndex`, etc

The React component names make sense in the context of where they show up within the viewer. Like if we were to make a change later on to let users pass in the child components of the `Linear/Circular` components, I think that the way things are now, below, is more intuitive than it would be after the change (down below)

```
<LinearViewer>
  <Index />
  <Annotations />
</LinearViewer>
```

vs

```
<LinearViewer>
  <Index />
  <AnnotationsViewer />
</LinearViewer>
```